build.sh

```diff
@@ -18,8 +18,8 @@ ARGS=$*
 # scripts, and that this script resides in the repo dir!
 REPODIR=$(cd $(dirname $0); pwd)
 
-VALIDARGS="clean libcuvs python rust docs tests examples --uninstall  -v -g -n --compile-static-lib --allgpuarch --no-nvtx --show_depr_warn --incl-cache-stats --time -h"
-HELP="$0 [<target> ...] [<flag> ...] [--cmake-args=\"<args>\"] [--cache-tool=<tool>] [--limit-tests=<targets>] [--build-metrics=<filename>]
+VALIDARGS="clean libcuvs python rust docs tests bench-ann examples --uninstall  -v -g -n --compile-static-lib --allgpuarch --no-nvtx --show_depr_warn --incl-cache-stats --time -h"
```

Comment:
Can we rename this to benchmarks, maybe? I'm thinking maybe we could start building one benchmarks suite but maybe have different entrypoints for the ANN grid searches vs clustering benchmarking, for example.

Reply:
This sounds like the benchmark infrastructure differs significantly between vector search vs clustering.
What about having multiple entry points but renaming them to better reflect what we're benchmarking? e.g. ~bench-search~, bench-clustering.
Update: bench-ann currently handles search and indexing, so probably "bench-search" is not very appropriate. Maybe keep this `bench-ann` and add the `bench-clustering`, `bench-dists` later?